tests/cases/py_client/cli_test.py

```diff
@@ -281,6 +281,8 @@ def checkParams(url, request):
                 args + ['--reference', 'reference_string'], username='mylogin', password='password')
 
         # Test if reference is sent with each file upload
+        fileList = os.listdir(localDir)
+        self.assertEqual(len(queryList), len(fileList))
```

Comment:
Just as one additional sanity check to assist us in the future, let's also assert that the list is not empty.